src/NzbDrone.Core/Music/AddArtistService.cs

```diff
@@ -44,7 +45,7 @@ public Artist AddArtist(Artist newArtist)
         {
             Ensure.That(newArtist, () => newArtist).IsNotNull();
 
-            newArtist = AddSkyhookData(newArtist);
+            //newArtist = AddSkyhookData(newArtist);
```

Comment:
I was testing to see if we could get rid of this first call to Skyhook during an artist add to prevent the double call to Metadata API, and let the RefreshArtistService continue to make the second call. We seem to already get everything we need from the initial artist search to add the artist.